index.html

```diff
@@ -149,14 +149,14 @@ <h3 class="projectName">Project name goes here</h3>
             </ul>
 
             <button class="bordersolid button2">See this project
-                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+                <img src="resources/Union.png" alt="Yet An Even Cooler Arrow Pointing Right">
             </button>
         </div>
     </section>
 
     <header class="dflex2 aboutTopMargin">
         <div>
-            <p class="p2">About</p>
+            <h1 class="p2">About</h1>
```

Comment:
- [ OPTIONAL ] I think it would be better if you update it to be h2 or h3 instead of h1 so you don't have issues with the accessibilty so kindly fix it 🙏🏻🔥